native/c/zcli.hpp

```diff
@@ -715,6 +716,7 @@ int ZCLI::parse(int argc, char *argv[])
       }
 
       rc = run(entries.size(), args);
+      cout << "\r[" << rc << "]" << flush;
```

Comment:
Could we use a TTY check to determine whether to print the RC? That would keep the same behavior for users in a shell while allowing us to perform different logic when we're communicating through the CLI/VSCE.